.dockerignore

```diff
@@ -0,0 +1,2 @@
+build/*
+.git
```

Comment:
Perhaps there is more we can ignore, but it's purely about performance rather than correctness (unless we are including some secret credentials in the image).